vmc/resource_vmc_cluster.go

```diff
@@ -108,6 +110,31 @@ func resourceCluster() *schema.Resource {
 				ValidateFunc: validation.IntBetween(3, 16),
 				Description:  "The maximum number of hosts that the cluster can scale out to.",
 			},
+			"microsoft_licensing_config": {
+				Type: schema.TypeList,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"mssql_licensing": {
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "The status of MSSQL licensing for this SDDC’s clusters. Possible values : ENABLED or DISABLED.",
+							ValidateFunc: validation.StringInSlice([]string{
+								LicenseConfigEnabled, LicenseConfigDisabled,
+							}, false),
+						},
+						"windows_licensing": {
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "The status of Windows licensing for this SDDC's clusters. Possible values : ENABLED, DISABLED or CUSTOMER'S ",
```

Comment:
What is the option CUSTOMER'S? If it is a valid option, why is it missing from the validatefunc()